src/components/Promise/index.js

```diff
@@ -42,7 +46,7 @@ function Promise({
           <RichTypography variant="h1" className={classes.promiseTitle}>
             {promise.title}
           </RichTypography>
-          <div className={classes.featuredImageContainer} />
+          <div className={classes.featuredImageContainer} {...props} />
```

Comment:
```suggestion
          <div className={classes.featuredImageContainer} />
```

Why are we passing props to this `div`?